src/main/java/com/zipcodewilmington/StringArrayUtils.java

```diff
@@ -9,99 +9,223 @@
      * @return first element of specified array
      */ // TODO
     public static String getFirstElement(String[] array) {
-        return null;
+        /*
+        Description
+        Given an array of String objects, return the first element of the array.
+         */
+
+        return array[0];
     }
 
+
+
     /**
      * @param array array of String objects
      * @return second element in specified array
      */
     public static String getSecondElement(String[] array) {
-        return null;
+        return array[1];
     }
 
+
+
     /**
      * @param array array of String objects
      * @return last element in specified array
      */ // TODO
     public static String getLastElement(String[] array) {
-        return null;
+        return array[array.length - 1];
     }
 
+
+
     /**
      * @param array array of String objects
      * @return second to last element in specified array
      */ // TODO
     public static String getSecondToLastElement(String[] array) {
-        return null;
+        return array[array.length - 2];
     }
 
+
+
     /**
      * @param array array of String objects
      * @param value value to check array for
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
-        return false;
+
+        boolean answer = false;
+
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == value) {
+                answer = true;
+                break;
+            }
+        }
+
+        return answer;
     }
 
+
+
     /**
      * @param array of String objects
      * @return an array with identical contents in reverse order
      */ // TODO
     public static String[] reverse(String[] array) {
-        return null;
+
+        String[] reversedArray = new String[array.length];
+
+        for (int i = 0; i < array.length; i++) {
+            reversedArray[(array.length - 1) - i] = array[i];
+        }
+
+        return reversedArray;
     }
 
+
+
     /**
      * @param array array of String objects
      * @return true if the order of the array is the same backwards and forwards
      */ // TODO
     public static boolean isPalindromic(String[] array) {
-        return false;
+
```

Comment:
```java
return Arrays.equals(reverse(array), array);
```

Reply:
Yes, that makes sense. I didn't think of that, but also I wanted to create a method that would consume less memory and time when the array was really long (millions of characters), by comparing the beginning and end of the string one by one and moving towards the center.